accelerator/BUILD

```diff
@@ -92,3 +65,42 @@ container_push(
     repository = "dltcollab/tangle-accelerator",
     tag = "latest",
 )
+
+cc_library(
+    name = "build_option",
+    defines = select({
+        ":mqtt_enable": ["MQTT_ENABLE"],
+        "//conditions:default": [],
+    }) + select({
+        ":db_enable": ["DB_ENABLE"],
+        "//conditions:default": [],
+    }) + select({
+        ":debug_mode": [],
+        "//conditions:default": ["NDEBUG"],
+    }),
+    visibility = ["//visibility:public"],
+)
+
+config_setting(
+    name = "debug_mode",
+    values = {
+        "define": "build_type=debug",
+    },
+)
+
+config_setting(
+    name = "profiling_mode",
```

Comment:
Shouldn't it be `profile_mode` instead of `profiling_mode`?

Reply:
Yes, It should be`profile_mode`.